### DISCUSSION TITLE
Switch github education benefits (teacher) from on e account to another

### DISCUSSION THREAD

**Discussion by draperez** [community/community#77836] [GitHub Education]
### Hi everyone,

I had a github account that I used for personal purposes. I started working in a university and I asked the educational benefits. It was accepted.

The problem came when I decided to create another account, separating personal and professional projects, so the email of the university is now in the professional account. Now I have no access to benefits (e.g. github copilot) in neither account.

Is there any way to solve the problem? Would removing my personal account solve the issue?

Thank you in advance
Alain

**Comment by dakshithadissanayaka** [comment]
Sure, here is a summary of the discussion and a possible solution:

**Issue:**

A user is asking about the process of switching GitHub Education benefits from one account to another.

**Possible Solution:**

Switching GitHub Education benefits from one account to another requires contacting GitHub Education support. They will be able to verify your eligibility and transfer the benefits to your new account. Here are the steps to follow:

1. **Gather Information:** Have your GitHub Education account details ready, including your old and new email addresses, associated student ID numbers, and any relevant documentation.

2. **Contact GitHub Education Support:** Open a support ticket on the GitHub Education help center. Briefly explain the situation and request to switch your GitHub Education benefits to your new account.

3. **Verify Eligibility:** GitHub Education support will verify your eligibility for the benefits and may ask for additional information or documentation.

4. **Benefit Transfer:** Once your eligibility is confirmed, GitHub Education support will process the transfer of benefits to your new account. This may take some time depending on the complexity of the case.

5. **Account Activation:** Once the transfer is complete, you will receive a confirmation email and should be able to activate your GitHub Education benefits on your new account.

Remember to be patient and provide accurate information to expedite the process.

**Reply by draperez** [reply]
Thank you for the answer!

Nevertheless, I have a problem with the solution. If I open https://support.github.com/contact/education and try to create a ticket, it says (translated from Spanish):

>  Individual manual reviews and status updates are no longer provided.

![2023-12-05 09-57-21](https://github.com/community/community/assets/63298434/39b144d9-1228-472e-b3ef-22e9bd53eca9)
![2023-12-05 09-59-53](https://github.com/community/community/assets/63298434/24c52e20-ed38-4d65-bf4a-df49f30594fa)

And the button is disabled with the text "Cannot open ticket for this request".
![2023-12-05 10-00-37](https://github.com/community/community/assets/63298434/d95cb92e-8e63-4229-bc6d-ed562b2c8c87)
